Python/pywarpx/picmi.py

```diff
@@ -3033,7 +3033,7 @@ def initialize_inputs(self):
 
         self.solver.solver_initialize_inputs()
 
-        # Initialize warpx.field_centering_no<x,y,z> and warpx.current_centering_no<x,y,z>
+        # Initialize warpx.field_centering_no<x,y,z> and warpx.m_current_centering_no<x,y,z>
```

Comment:
```suggestion
        # Initialize warpx.field_centering_no<x,y,z> and warpx.current_centering_no<x,y,z>
```

I think that, since we are in picmi, reverting this change would be better for the user.